be-scala-play/files/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/openjdk-11-runtime:1.15
+FROM registry.access.redhat.com/ubi9/openjdk-11-runtime:latest
```

Comment:
@braisvq1996 I would recommend to not go for `latest` since it could break builds and have unforeseen effects.

Reply:
The Java runtime will be 11 even if we use latest tag so there should not be any issue in this case.